components/molecules/UserProfile.tsx

```diff
@@ -0,0 +1,67 @@
+import { MoonStar, Sun, User2 } from '@tamagui/lucide-icons'
+import React from 'react'
+import { useColorScheme } from 'react-native'
+import { Avatar, Button, Card, Text, View } from 'tamagui'
+
+import getColors from '~/constants/Colors'
+import { RADIUS_BUTTON } from '~/constants/Constants'
+import { type Colors } from '~/interfaces/Colors'
+import type User from '~/interfaces/User'
+
+interface props {
+  user: User
+}
+
+const UserProfile = ({ user }: props): JSX.Element => {
+  const isDarkMode = useColorScheme() === 'dark'
+  const colors: Colors = getColors(useColorScheme())
+
+  return (
+    <Card
+      gap="$4"
+      backgroundColor={colors.lightMist}
+      paddingHorizontal="$4"
+      paddingVertical="$4"
+      flexDirection="row"
+      alignItems="center"
+      borderRadius={RADIUS_BUTTON}
+      elevation={8}
+      shadowColor="black"
```

Comment:
Use color in constants file

Reply:
@truongtdph40209  Those colors have already been defined by the `Tamagui` library, so there’s no need to redefine them